### PR TITLE
add deactivate user page

### PR DESCRIPTION
**Pull request by dtaylor7**
- add new page to allow admin to deactivate user
 - dont allow user to login if they are not active

### CODE REVIEW

**Comment by mateen-hussain** [comment]
@dtaylor7 Need few clarifications:

- Will all new users have `is_active` flag set to True ?
- Does that mean, we will have to retrospectively set this flag to true for existing users?
- I was thinking rather than `is_active` maybe it should be `is_deactivated` as it would involve minimal changes?

**Comment by mateen-hussain** [comment]
Will the deactivated user be shown on user list page ? And if so will deactivate button be disabled for the user. Maybe this is our of scope for this ticket, if so then good to create a ticket in backlog.

File: pages/admin/user-management/deactivate/DeactivateUser.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  get middleware() {
+    return [
+      ...authentication.protect(['admin']),
+      flash
+    ];
+  }
+
+  async getUser() {
+    return await User.findOne({
```
Comment (mateen-hussain) [inline pages/admin/user-management/deactivate/DeactivateUser.js:29]: await not needed :D

Comment (dtaylor7) [reply pages/admin/user-management/deactivate/DeactivateUser.js:29]: i have thought about this and i believe its useful to keep the await ( although not strictly needed ). The reason is that it documents that the function we are calling is a promise or async await function. If we remove this a future developer would need to need lookup the user.findOne function to see if its a promise or not

**Review by mateen-hussain — APPROVED** [review]

**Pull request merged**